setup.py

```diff
@@ -32,6 +32,8 @@
     "openapi-core==0.13.8",
     "zope.sqlalchemy==1.5",
     "pydantic==1.8.2",
+    "ghga-service-chassis-lib==0.3.0",
```

Comment:
```suggestion
    "ghga-service-chassis-lib[pubsub]==0.3.0",
```

I think we need the pubsub extra don't we?

Reply:
Yes. It was an attempt at a fix on my end.